src/components/UI/Display/Cluster/ClusterIDLabel/index.tsx

```diff
@@ -71,7 +71,7 @@ const ClusterIDLabel: React.FC<IClusterIDLabelProps> = ({
       <OverlayTrigger
         overlay={
           <Tooltip id='idtooltip'>
-            <>Cluster ID: {clusterID}</>
+            <>Cluster name: {clusterID}</>
```

Comment:
I'm sorry for not giving more context in the issue 😅 

In the legacy GS API features we use the `Cluster ID` naming, and in the new K8s Management API features  we call them `Cluster name`. It's technically the same thing, but for the MAPI implementations we aim to be closer to the terminology of K8s resources.

> More on K8s resource naming:
> https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names

That being said, the `ClusterIDLabel` component is used by both GS API and MAPI worlds, so just changing `ID` to `Name` doesn't work, because it wouldn't make sense any more in the legacy world (which is what our customers currently use).

Ideally there should be a way to configure which label is shown, either by specifying the label directly as a prop (`label='Name'`), or which variant of the component to use - with an enum (`variant={ClusterIDLabelType.Name}`).

Reply:
Got it - thanks for clarifying!